pixeltable/catalog/table_version.py

```diff
@@ -1508,6 +1520,16 @@ def version(self) -> int:
     def version(self, version: int) -> None:
         assert self.effective_version is None
         self._tbl_md.current_version = version
+        self._version_md.version = version
+
+    @property
+    def created_at(self) -> float:
+        return self._version_md.created_at
+
+    @created_at.setter
+    def created_at(self, ts: float) -> None:
+        assert self.effective_version is None
+        self._version_md.created_at = ts
```

Comment:
I don't love the in-place update here; I'd prefer these be frozen dataclasses and updated via

`self._version_md = self._version_md.replace(created_at=ts)`

Reply:
Why?